<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-GOMK6OD" name="Test" tests="4" failures="1" timestamp="27 Jul 2018 10:50:11 GMT" time="180.497" errors="0">
  <testcase name="RestTest1" time="143.48" classname="Rest.restRunner"/>
  <testcase name="SoapTest" time="35.057" classname="Soap.testRunner">
    <failure type="java.lang.AssertionError" message="expected [FAILED] but found [FINISHED]">
      <![CDATA[java.lang.AssertionError: expected [FAILED] but found [FINISHED]
	at Soap.testRunner.SoapTest(testRunner.java:44)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- SoapTest -->
  <testcase name="singleTest" time="0.256" classname="Soap.testRunner2"/>
  <testcase name="RestTest2" time="1.536" classname="Rest.restRunner2"/>
</testsuite> <!-- Test -->
